ReadMe.md

```
@@ -170,47 +186,68 @@ python main.py openai.find_closest_match --search_param="SEARCH_TERM" --options=
-python main.py google_sheets.sheet_to_create_time_logs_from_data --title="SHEET_TITLE" --spreadsheet_id="YOUR_SPREADSHEET_ID" --dry_run=True
+python main.py google_sheets sync_sheet_to_wrike --title="SHEET_TITLE" --spreadsheet_id="YOUR_SPREADSHEET_ID" --dry_run=True
 ```
 
-Replace `YOUR_SPREADSHEET_ID` and `SHEET_TITLE` with the appropriate values. Use `--dry_run=True` for testing without making changes.
+Replace `YOUR_SPREADSHEET_ID` and `SHEET_TITLE` with the appropriate values in each command. Use `--dry_run=True` for testing mode to validate data without making changes in the target platform.
+
 
 #### Jira
 
-Log time to a Jira task:
+###### Log time to a Jira task
```

Markdownlint

Expected: h5; Actual: h6
Heading levels should only increment by one level at a time

(MD001, heading-increment)